src/Core/Components/SortableList/FluentSortableList.razor.js

```diff
@@ -0,0 +1,34 @@
+export function init(id, group, pull, put, sort, handle, filter, component) {
```

Comment:
Is it possible to test if the `Sortable` class exists, to try to download the latest version of the JS file `Sortable.min.js`?

In any case, you need to test the existence of this class, because if your JS hasn't had time to be downloaded, it will trigger an error.

Reply:
Yes, will work on that